gradio/event_queue.py

```diff
@@ -246,7 +250,7 @@ async def process_event(cls, event: Event) -> None:
             json=event.data,
         )
         end_time = time.time()
-        cls.update_estimation(duration=round(end_time - begin_time, 3))
+        cls.update_estimation(end_time - begin_time)
```

Comment:
why did you drop the round?
I did add that to not send floats with 10 digits after the "." to the frontend. 

- decreases package size
- makes it more readable with few digits

Reply:
rounding should be done at the last step in the front end. Rounding at every step in a calculation introduces compounding errors and is unnecessary calculation. for example:
rounding done at the last step: 0.44 + 0.44 = 0.88. round(0.88, 1) => 0.9
rounding done at every step: round(0.44, 1) + round(0.44, 1) => 0.8